packages/client/src/__tests__/integration/happy/disconnect-while-query/test.ts

```diff
@@ -6,10 +6,13 @@ test('disconnect-while-query', async () => {
 
   await prisma.user.findMany()
   const a = prisma.user.findMany()
-  prisma.$disconnect()
 
-  await a
+  prisma.$disconnect()
+  await a // re-wakes the engine
   await prisma.$disconnect()
+  await a // re-wakes the engine
 
   expect(await a).toMatchInlineSnapshot(`Array []`)
+
+  await prisma.$disconnect()
```

Comment:
This tests was blocking the CI/jest because doing `await` on the prisma promise re-wakes the engine, even afyter a disconnect - which is the purpose of the test. However, the test did not do a final `$disconnect` after doing that.

Reply:
The test now does an additinoal `await a` in line 13 - is that necessary if line 15 does that in the `expect` was well?

Reply:
Indeed not needed.